tests/test_worktree_management.py

```
@@ -193,3 +193,128 @@ def test_list_worktrees_empty_when_none_exist(tmp_path):
         assert res.stdout.strip() == ""  # Empty output
     finally:
         os.chdir(original_dir)
+
+
+def _init_bare_repo(bare_path: Path):
```

Ruff (0.15.0)

[warning] Missing return type annotation for private function `_init_bare_repo`

Add return type annotation: `None`

(ANN202)